src/globus_sdk/experimental/globus_app/_validating_token_storage.py

```diff
@@ -2,288 +2,124 @@
 
 import typing as t
```

Comment:
`validating_token_storage` should be a dedicated module if we add two support modules.

Something like
```
  globus_app/
    |- app.py
    |- validating_token_storage/
      |- __init__.py
      |- token_validators.py
      |- identity_id_reader.py
      |- token_storage.py
```